src/groundStation/system.py

```diff
@@ -76,6 +76,61 @@ def __init__(self):
             'LAST': 31
         }
         self.SERVICES = {
+            'SCHEDULER': {
+                'port': 15,  # share a port with EPS time service
+                # TODO: these need a error response value
+                'subservice': {
+                    'SET_SCHEDULE': {
+                        'subPort': 0,
+                        'inoutInfo': {
+                            'args': None,  # All scheduled commands should be stored in schedule.txt
+                            'returns': {
+                                'err': '>b', # Returns 0 on success. Refer for schedule.h for calloc error code. Refer to rederrno.h for reliance edge error codes
+                                'number of cmds scheduled': '>b'    #Returns the number of cmds left in the schedule file. Returns -1 if an error occurred. 
```

Comment:
There's a 'What' key that allows you to place these comments. They will then get transferred to the autogenerated docs

Reply:
@yunqingyi

Reply:
fixed